libqtile/ipc.py

```diff
@@ -107,18 +107,18 @@ def pack(msg: Any, *, is_json: bool = False) -> bytes:
 
 
 class Client:
-    def __init__(self, fname: str, is_json=False) -> None:
+    def __init__(self, socket: str, is_json=False) -> None:
         """Create a new IPC client
 
         Parameters
         ----------
-        fname : str
+        socket : str
             The file path to the file that is used to open the connection to
             the running IPC server.
         is_json : bool
             Pack and unpack messages as json
         """
-        self.fname = fname
+        self.socket = socket
```

Comment:
How about socket_path instead, since this isn't the actual socket?

Reply:
fair, will just require some reformatting